lib/tasks/fill_record_table.rake

```diff
@@ -9,7 +9,7 @@ namespace :records do
     ].each do |records|
       Result::MARKERS.compact.each do |marker|
         is_cr = RegionalRecord::CONTINENT_TO_RECORD_MARKER.value?(marker)
-        record_scope = is_cr ? :CR : marker.to_sym
+        record_scope = is_cr ? :continental : { "WR" => :world, "NR" => :national }[marker]
```

Comment:
This mapping of "human-readable tags" to enum constant values could be moved into a constant